client/src/components/Meetup/Meetup.js

```diff
@@ -1,8 +1,689 @@
+          >
+            Cancel Join Request
+          </Button>
+        );
+        break;
+      case "rejected":
+        button = (
+          <Grid container spacing={1} direction="column" align="center">
+            <Grid item>
+              <Button startIcon={<PersonAddIcon />} disabled variant="outlined">
+                Join Meetup
+              </Button>
+            </Grid>
+            <Grid item>
+              <Typography variant="body2" className={classes.error}>
+                you were rejected from the meetup.
+              </Typography>
+            </Grid>
+          </Grid>
+        );
+        break;
+      default:
+        button = (
+          <Button
+            startIcon={<PersonAddIcon />}
+            onClick={handleJoinMeetup}
+            disabled={isUpdating}
+          >
+            Join Meetup
+          </Button>
+        );
+    }
+    return (
+      <Grid item xs={2} container justify="center" alignItems="flex-start">
+        {button}
+      </Grid>
+    );
+  };
+
+  const renderMeetup = (errorStatus) => {
+    if (errorStatus || !eventDetails) {
+      return renderError(errorStatus);
+    }
+
+    return (
+      <Grid item container spacing={2}>
+        <Grid item container>
+          <Grid item xs>
+            <Typography variant="h3">{eventDetails.title}</Typography>
+            {/* add share/link copy icon here? */}
+          </Grid>
+          {renderMeetupAction()}
+        </Grid>
+        <Grid
+          item
+          container
+          direction="column"
+          spacing={1}
+          className={classes.meetupDetails}
+        >
+          <Grid item>
+            <Typography variant="body2">
+              Time:{" "}
+              {eventDetails.time.toLocaleString(locale, meetupTimeOptions)}
+            </Typography>
+          </Grid>
+          {renderLocation()}
+          <Grid item>
+            <Typography variant="body2">
+              # Attendees:{" "}
+              {`${attendees ? attendees.length : 0} out of ${
+                eventDetails.maxCapacity
+              } (min. ${eventDetails.minCapacity})`}
+            </Typography>
+          </Grid>
+          <Grid item>
+            <Typography
+              component="ul"
+              className={classes.tagList}
+              variant="body2"
+            >
+              Tags: {eventDetails.tags ? renderTags(eventDetails.tags) : "none"}
+            </Typography>
+          </Grid>
+          <Grid item>
+            <Typography className={classes.description} variant="body1">
+              {eventDetails.description}
+            </Typography>
+          </Grid>
+          {renderOrganizer()}
+          <Grid item container justify="center" spacing={1}>
+            {renderAttendees()}
+            {user &&
+              user.id === eventDetails.owner.id &&
+              renderPendingAttendees(pendingAttendees)}
+          </Grid>
+        </Grid>
+      </Grid>
+    );
+  };
+
+  return (
```

Comment:
Please use a `Container` to wrap everything. Set `maxWidth` to either `"sm"` or `"md"` depending on what looks better.